stretch-auth/src/FormContainer/FormContainer.js

```diff
@@ -55,62 +55,65 @@ class FormContainer extends Component {
-          <p className='release-sprite-text'>Click To Release</p>
-          </div>
-        ) ||
-          <PokeForm allPokemon={this.props.allPokemon} showPokemon={this.showPokemon} slot={"slot1"}/>
-        }
-        {(this.state.slot2.pokemon &&
-          <div className='poke-sprite-div'>
-          <p className='poke-sprite-name'>{this.state.slot2.name}</p>
-          <img className='poke-sprite' src={this.state.slot2.sprite} alt={`${this.state.slot2.pokemon} sprite`} onClick={() => this.removePokemon('slot2')}/>
-          <p className='release-sprite-text'>Click To Release</p>
-          </div>
-        ) ||
-          <PokeForm allPokemon={this.props.allPokemon} showPokemon={this.showPokemon} slot={"slot2"}/>
-        }
-        {(this.state.slot3.pokemon &&
-          <div className='poke-sprite-div'>
-          <p className='poke-sprite-name'>{this.state.slot3.name}</p>
-          <img className='poke-sprite' src={this.state.slot3.sprite} alt={`${this.state.slot3.pokemon} sprite`} onClick={() => this.removePokemon('slot3')}/>
-          <p className='release-sprite-text'>Click To Release</p>
-          </div>
-        ) ||
-          <PokeForm allPokemon={this.props.allPokemon} showPokemon={this.showPokemon} slot={"slot3"}/>
-        }
-        {(this.state.slot4.pokemon &&
-          <div className='poke-sprite-div'>
-          <p className='poke-sprite-name'>{this.state.slot4.name}</p>
-          <img className='poke-sprite' src={this.state.slot4.sprite} alt={`${this.state.slot4.pokemon} sprite`} onClick={() => this.removePokemon('slot4')}/>
-          <p className='release-sprite-text'>Click To Release</p>
-          </div>
-        ) ||
-          <PokeForm allPokemon={this.props.allPokemon} showPokemon={this.showPokemon} slot={"slot4"}/>
-        }
-        {(this.state.slot5.pokemon &&
-          <div className='poke-sprite-div'>
-          <p className='poke-sprite-name'>{this.state.slot5.name}</p>
-          <img className='poke-sprite' src={this.state.slot5.sprite} alt={`${this.state.slot5.pokemon} sprite`} onClick={() => this.removePokemon('slot5')}/>
-          <p className='release-sprite-text'>Click To Release</p>
-          </div>
-        ) ||
-          <PokeForm allPokemon={this.props.allPokemon} showPokemon={this.showPokemon} slot={"slot5"}/>
-        }
-        {(this.state.slot6.pokemon &&
-          <div className='poke-sprite-div'>
-          <p className='poke-sprite-name'>{this.state.slot6.name}</p>
-          <img className='poke-sprite' src={this.state.slot6.sprite} alt={`${this.state.slot6.pokemon} sprite`} onClick={() => this.removePokemon('slot6')}/>
-          <p className='release-sprite-text'>Click To Release</p>
-          </div>
-        ) ||
-          <PokeForm allPokemon={this.props.allPokemon} showPokemon={this.showPokemon} slot={"slot6"}/>
-        }
-      </section>
+      <div>
+        <h3 className="error-message">{this.state.error}</h3>
```

Comment:
Good call on the error-message. Need to uncomment to lines on the last pull for testing it